controller/model/enrollment_mod_erott.go

```diff
@@ -143,7 +142,7 @@ func (module *EnrollModuleEr) Process(context EnrollmentContext) (*EnrollmentRes
 	}
 
 	cltFp := module.fingerprintGenerator.FromPem(cltPem)
-
+	cltFp = strings.Replace(strings.ToLower(cltFp), ":", "", -1)
```

Comment:
is there a reason this logic wasn't moved into fingerprintGenerator?

Reply:
fixed